pgnReader.py

```diff
@@ -0,0 +1,38 @@
+'''
+    path - path to .pgn file
+    maxGame - maximum number of games to parse

```

Comment:
What about naming this parameter max_num_games? It limits number of games, not a single game